packages/auth0-ai/auth0_ai/credentials.py

```diff
@@ -1,4 +1,7 @@
-from typing import TypedDict, Optional
+from typing import TypedDict, Optional, List, Required, ReadOnly
+
+class AuthorizationDetails(TypedDict):
+    type: Required[ReadOnly[str]]
```

Comment:
I'm getting an error saying that `ReadOnly` cannot be imported while using the version of python that I'm testing (v3.12). Again, we want to preserve compatibility with 3.11+, so we can remove usage of `ReadOnly` and that should be good enough.